x/stakeibc/keeper/keeper.go

```diff
@@ -108,17 +108,6 @@ func (k Keeper) GetAuthority() string {
 	return k.authority
 }
 
-// Searches all interchain accounts and finds the connection ID that corresponds with a given port ID
-func (k Keeper) GetConnectionIdFromICAPortId(ctx sdk.Context, portId string) (connectionId string, found bool) {
```

Comment:
For ICA controllers I see how the port is unique but what about for transfer channels where the port is always 'transfer'
How was this reverse lookup function working before?